src/audio/copier/copier.c

```diff
@@ -208,82 +208,63 @@ static int create_host(struct comp_dev *parent_dev, struct copier_data *cd,
 		       const struct ipc4_copier_module_cfg *copier_cfg,
 		       int dir)
 {
-	struct sof_uuid host = {0x8b9d100c, 0x6d78, 0x418f, {0x90, 0xa3, 0xe0,
-			0xe8, 0x05, 0xd0, 0x85, 0x2b}};
 	struct ipc_config_host ipc_host;
-	const struct comp_driver *drv;
-	struct comp_dev *dev;
 	struct host_data *hd;
 	int ret;
 
-	drv = ipc4_get_drv((uint8_t *)&host);
-	if (!drv)
-		return -EINVAL;
+	enum sof_ipc_frame __sparse_cache in_frame_fmt, out_frame_fmt;
```

Comment:
remove newline above

Reply:
this is still not addressed. Please remove newline on line 214

Reply:
done